sormas-api/src/main/resources/enum.properties

```diff
@@ -952,12 +952,14 @@ TaskType.CASE_MANAGEMENT = case management
 TaskType.CONTACT_FOLLOW_UP = contact follow up
 TaskType.CONTACT_INVESTIGATION = contact investigation
 TaskType.CONTACT_TRACING = contact tracing
+TaskType.CONTINUE_INVESTIGATION = continue the investigation following a change in the signal
```

Comment:
Rename to EVENT_CONTINUE_INVESTIGATION (for developer clarification), also please change "signal" to "event" because that's the general term (and signal is a specific event status)